test/correctness/constraints.cpp

```diff
@@ -8,7 +8,7 @@ using namespace Halide;
 
 bool error_occurred = false;
 void my_error_handler(void *user_context, const char *msg) {
-    printf("%s\n", msg);
+    //printf("%s\n", msg);
```

Comment:
Intended to commit this change?

Reply:
Yes, it's too noisy with it, but when debugging the test you want to uncomment it.